### PR TITLE
Starting User Profile DIsplay

### PR DESCRIPTION
**Pull request by jeeheezy**

### CODE REVIEW

**Review by CarlosCastanedaDev — APPROVED** [review]
Looks good Jeeho, great job!

File: app/views/photos/_photo.html.erb [hunk 1]
```diff
@@ -1,27 +1,59 @@
...
+      <div class="card-body">
+        <h5 class="card-title"><%= photo.owner.username %></h5>
+        <p class="card-text"><%= photo.caption %></p>
+      </div>
+      <ul class="list-group list-group-flush">
+        <% photo.comments.each do |comment| %>
+          <li class="list-group-item">
+            <div class="d-flex">
+              <div class="flex-shrink-0">
+                <img src="..." alt="...">
```
Comment (wahnahbe) [inline app/views/photos/_photo.html.erb:14]: maybe add a place holder but its just being picky

File: app/views/photos/_photo.html.erb [hunk 2]
```diff
@@ -1,27 +1,59 @@
...
+<!--  </p>
 
   <p>
     <strong>Likes count:</strong>
-    <%= photo.likes_count %>
-  </p>
+    <%#= photo.likes_count %>-->
+<!--  </p>
 
   <p>
```
Comment (wahnahbe) [inline app/views/photos/_photo.html.erb:48]: same as before comments okay just be careful of pushing

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -5,7 +5,7 @@
       <span class="navbar-toggler-icon"></span>
     </button>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
-      <ul class="navbar-nav mr-auto">
+      <ul class="navbar-nav mr-auto me-auto mb-2 mb-lg-0">
```
Comment (wahnahbe) [inline app/views/shared/_navbar.html.erb:8]: creative choice?

**Pull request merged**